src/types/guild.ts

```diff
@@ -660,3 +666,40 @@ export interface EditGuildTemplate {
   /** description for the template (0-120 characters) */
   description?: string | null;
 }
+
+/** */
+export interface MembershipScreeningPayload {
+  /** when the fields were last updated */
+  version: string;
+  /** the steps in the screening form */
+  form_fields: MembershipScreeningFieldPayload[];
+  /** the server description shown in the screening form */
+  description: string | null;
+}
+
+/** */
```

Comment:
?

Reply:
The "Document Membership Screening" PR is not yet merged, so the URLs are not available.